extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -31,6 +31,11 @@ static public function onGetRailModuleList( &$modules ) {
 		return true;
 	}
 
+	public static function onBeforePageDisplay( OutputPage $out, Skin $skin ) {
+		Wikia::addAssetsToOutput( 'recirculation_scss' );
```

Comment:
We could load the CSS along with the AJAX query fetching the fandom posts rather than loading this eagerly.